app/src/main/java/com/eventyay/organizer/data/order/Statistics.java

```diff
@@ -27,7 +27,7 @@
     public Long cancelled;
     public Long pending;
     public Long placed;
-    public Long total;
+    public long total;
```

Comment:
Why?

Reply:
Check this [link](https://stackoverflow.com/questions/42872201/data-binding-safeunbox-warning) . I followed the same thing in one of the earlier P.R as well where the same problem was addressed.